src/bank/use-cases/find-bank-by-id.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@nestjs/common';
+
+import { Bank } from '@/bank/infra/entities';
+import { IBankRepository } from '@/bank/interfaces';
+
+@Injectable()
+export class GetBankByIdUseCase {
```

Comment:
Padronizar nome da classe com o nome do arquivo 
```suggestion
export class FindBankByIdUseCase {
```